src/ui-kit/atoms/DeleteEdit/DeleteEdit.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react"
+
+import closeButton from "uikit/icons/close-button.svg"
+import deleteEdit from "uikit/icons/delete-edit.svg"
+import "./DeleteEdit.scss"
+
+type Props = {
+    deleteAction?: any;
+    editAction?: any;
+    id?: string;
+}
+
+export const DeleteEdit: React.FC<Props> = (props: Props) => {
+    const { deleteAction, editAction, id } = props
+    const [isOpen, setIsOpen] = React.useState(false)
+    const toggle = () => setIsOpen(!isOpen);    
```

Comment:
if you'd like to change state depending on the previous state - access the previous value using the first param in setter:
```suggestion
    const toggle = () => setIsOpen(open => !open);
```